### PR TITLE
Remove examples dir issue #584

### PR DESCRIPTION
**Pull request by DanTanAtAims**
Completing Issue #584 

Removed all the scripts in the examples directory as they were already in the cookbook documentation (almost identical).

 Optional Changes that I can revert (revert to commit [09b6daa](https://github.com/open-AIMS/ADRIA.jl/commit/09b6daa09285c8caab21bcd14e65f2416e9e902d)).

Moved the data in examples into the test/data directory as that was the only placed that is using it and is not referenced in the cookbook. Updated testing scripts and index.md to reflect this move.

Updated index.md to point to cookbook. Unsure if the documentation links are working.

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -37,6 +37,6 @@ Here are some answers to some issues encountered.
 
 **A.** Scenarios are defined in a CSV file (with parameter values in columns, so that each row defines a scenario).
 
-- See the `example_scenarios.csv` file in the `examples` directory for an idea of what this looks like.
-- See `parameters.jl` file in the `examples` directory on how to extract the model specification and parameter table for a given domain.
-- See also the `running_scenarios.jl` example script which showcases how to run such a file for a given study area.
+- See the `example_scenarios.csv` file in the `test/data` directory for an idea of what this looks like.
+- See `Extratcing model details` example in [Cookbook examples](@ref) on how to extract the model specification and parameter table for a given domain.
```
Comment (ConnectedSystems) [inline docs/src/index.md:41]:
```suggestion
- See `Extracting model details` example in [Cookbook examples](@ref) on how to extract the model specification and parameter table for a given domain.
```

File: test/runtests.jl [hunk 2]
```diff
@@ -9,8 +9,7 @@ using ADRIA.metrics: total_absolute_cover
 const ADRIA_DIR = pkgdir(ADRIA)
 const TEST_DATA_DIR = joinpath(ADRIA_DIR, "test", "data")
 
-const EXAMPLES_PATH = joinpath(ADRIA_DIR, "examples")
-const EXAMPLE_DOMAIN_PATH = joinpath(EXAMPLES_PATH, "Test_domain")
+const EXAMPLE_DOMAIN_PATH = joinpath(TEST_DATA_DIR, "Test_domain")
```
Comment (ConnectedSystems) [inline test/runtests.jl:12]: Best to rename this const as `TEST_DOMAIN_PATH` I think

File: test/seeding.jl [hunk 3]
```diff
@@ -7,7 +7,7 @@ using ADRIA: distribute_seeded_corals, location_k, seed_corals!
 
 @testset "Seeding" begin
     # first test function on example domain
-    dom = ADRIA.load_domain(joinpath(@__DIR__, "..", "examples", "Test_domain"), 45)
+    dom = ADRIA.load_domain(joinpath(@__DIR__, "data", "Test_domain"), 45)
```
Comment (ConnectedSystems) [inline test/seeding.jl:10]: This should be updated to `TEST_DOMAIN_PATH` for consistency

File: test/runtests.jl (hunk 2 above)
Comment (ConnectedSystems) [reply test/runtests.jl:12]: Be sure to go through and update all occurrences of `EXAMPLE_DOMAIN_PATH`

File: docs/src/index.md [hunk 4]
```diff
@@ -37,6 +37,6 @@ Here are some answers to some issues encountered.
 
 **A.** Scenarios are defined in a CSV file (with parameter values in columns, so that each row defines a scenario).
 
-- See the `example_scenarios.csv` file in the `examples` directory for an idea of what this looks like.
-- See `parameters.jl` file in the `examples` directory on how to extract the model specification and parameter table for a given domain.
-- See also the `running_scenarios.jl` example script which showcases how to run such a file for a given study area.
+- See the `example_scenarios.csv` file in the `test/data` directory for an idea of what this looks like.
```
Comment (ConnectedSystems) [inline docs/src/index.md:40]: Hmm, I'm thinking we link to the file in the data-specs [repository](https://github.com/open-AIMS/ADRIA-data-specs) instead as users won't have direct access to the `test/data` directory when installed, and referring to the `test` directory is a bit awkward.

Note that the `example_scenarios.csv` file there is horribly outdated. The copy we have here is more up to date, so please submit a PR to update the example scenarios file in the data-specs repo.

**Comment by ConnectedSystems** [comment]
Hi @DanTanAtAims 

This is looking good, requesting a few minor changes.

Unrelated to the PR:

1. Please mention the issue ID in the description rather than the title
2. Use [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) in the description

e.g., instead of "Completing issue [issue number]", state "closing [issue number]"

**Comment by ConnectedSystems** [comment]
Pinging @Rosejoycrocker in case this affects any recent/ongoing changes regarding the test data

File: docs/src/index.md (hunk 4 above)
Comment (DanTanAtAims) [reply docs/src/index.md]: I have a push ready for the data specification repo currently don't have the permissions. 

Did you want to update just the example_scenarios.csv?

Comment (ConnectedSystems) [reply docs/src/index.md]: Ah sorry, I'll give you access tomorrow morning!

Comment (ConnectedSystems) [reply docs/src/index.md]: You should have access now.

And yes, just the `example_scenarios.csv` to be updated. Thank you!

**Comment by DanTanAtAims** [comment]
@ConnectedSystems I've address the comments in the newest [commit](https:\\github.com\open-Aims\ADRIA.jl\commit\e7178abb23d3f7ec666b6ac87dfae7f8bbdba530)

**Review by ConnectedSystems — APPROVED** [review]
Closes #584

**Comment by ConnectedSystems** [comment]
I'm on the road shortly but wanted to check @DanTanAtAims you ran tests locally and it all passes?

**Pull request merged**